frontend/src/components/Question/_ButtonArray.scss

```diff
@@ -12,7 +12,18 @@
 
     .btn-group-toggle-custom {
         display: flex;
-        column-gap: 3rem;
+        &:has(> :last-child:nth-child(2)) {
+            column-gap: 8vw;
+        }
+        &:has(> :last-child:nth-child(3)) {
+            column-gap: 6vw;
+        }
+        &:has(> :last-child:nth-child(4)) {
+            column-gap: 3vw;
+        }
+        &:has(> :last-child:nth-child(5)) {
+            column-gap: 2vw;
+        }
```

Comment:
I don't know if you had considered it, but it's also possible to just do `justify-content: space-between` for automatic spacing between buttons. https://www.w3schools.com/cssref/playdemo.php?filename=playcss_justify-content

Reply:
I think I experimented with that before, but then ended up with spacing that was sup-optimal, hence switched to column-gap. But I'll give it another try, thanks!